testsuite/README.md

```diff
@@ -57,6 +57,9 @@ no matter their value, or lack of one.
 - `ALIRE_DISABLE_NETWORK_TESTS`: when defined, tests that
   require non-local network use will be skipped.
 
+- `ALIRE_ENABLE_LOCAL_TESTS`: when defined, tests that are intended to be run
```

Comment:
I think it would be better to prefix this testsuite variables with `ALIRE_TESTSUITE_` instead of `ALIRE_`

Reply:
Indeed.